alluxio/pom.xml

```diff
@@ -47,6 +47,7 @@
         <dependency>
               <groupId>com.google.guava</groupId>
               <artifactId>guava</artifactId>
+              <version>15.0</version>
```

Comment:
We should not put guava in zeppelin-interpreter as each interpreter may need different versions of guava. E.g. Pig needs 15.0 while Python needs 18.0